lxd/device/device_utils_unix.go

```diff
@@ -170,18 +184,28 @@ func UnixDeviceCreate(s *state.State, idmapSet *idmap.IdmapSet, devicesPath stri
 		d.Type = "c"
 	}
 
-	// Get the device owner.
-	if m["uid"] != "" {
-		d.UID, err = strconv.Atoi(m["uid"])
+	if shared.IsTrue(m["ownership.inherit"]) {
```

Comment:
Shall we tweak this so that ` unixDeviceOwnership` is run when either m["uid"] or m["gid"] are unset, and then only set the key(s) that aren't explicitly overridden?